go/vt/vterrors/last_error.go

```diff
@@ -87,7 +86,7 @@ func (le *lastError) shouldRetry() bool {
 		// within the max time range
 		return true
 	}
-	log.Errorf("VReplication encountered the same error continuously since %s, we will assume this is a non-recoverable error and will not retry anymore; the workflow will need to be manually restarted once error '%s' has been addressed",
-		le.firstSeen.UTC(), le.err)
+	log.Errorf("%s: the same error was encountered continuously since %s, it is now assumed to be unrecoverable; any affected workflows will need to be manually restarted once error '%s' has been addressed",
```

Comment:
Should we say operations instead of workflows?

Reply:
fixed.